helm-chart/binderhub/values.yaml

```diff
@@ -192,7 +192,6 @@ jupyterhub:
     cmd: jupyter-notebook
     # use jupyterlab for the default UI
     defaultUrl: /lab
-    events: false
```

Comment:
Do you know why this default value is removed? These are the default values for the helm charts and I'd like there to be an entry for all valid configurations (unless its passthrough config).

Reply:
Yes, these are the events this PR is relaying to the user, so disabling them mostly defeats the purpose. I was thinking about this, and maybe the right thing to do is make this explicitly true here, rather than relying on the upstream chart default, which happens to be true.

Reply:
Oh I didn't realize this was a z2jh config. Then it was acceptable to remove it, but I think indeed it should be set to true here so z2jh can modify it without breaking this, and so that we understand that binderhub requires it to be true for this to function.

Well hmm, I assume this won't break if events are disabled though, right? And I figure that the `events` serve the exact same purpose in z2jh as in binderhub, so I think it can be removed after all --- assuming it won't break something causing errors etc.

Reply:
It won't break - both work, just you get the existing PR if this is disable,d and you get more info if it's enabled. I think it's sensible for binderhub to set this to true explicitly, though, even if that happens to be the current default on the upstream chart. Either way is fine by me.